.github/workflows/gh-pages-deployment.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - redat97/issue24
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+          cache-dependency-path: 'ai-cfia-ia-acia'
+
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+        working-directory: ai-cfia-ia-acia
+
+      - name: Build website
+        run: yarn build
+        working-directory: ai-cfia-ia-acia
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./ai-cfia-ia-acia/build
+          publish_branch: deployment
+
+          user_name: github-actions[bot]
+          user_email: 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
can you explain the choice of email?

Reply:
it's default github action bot email
https://github.com/marketplace/actions/bot-details 
and is used in the example in docusaurus documentation
https://docusaurus.io/docs/deployment#triggering-deployment-with-github-actions

Reply:
can you put this information in a comment in the file? usually if I ask for clarification it is meant for you to clarify the code.

Reply:
ah yeah make sense mb

Reply:
don't just put a end-of-line comment; create a real comment with the links you posted here

Reply:
done